plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -1190,7 +1213,7 @@ def update_auth_policy_server(self, ipAddress):
             if is_ise_server:
                 trusted_server = self.want.get("trusted_server")
                 self.accept_cisco_ise_server_certificate(ipAddress, trusted_server)
-                time.sleep(15)
+                time.sleep(20)
```

Comment:
Please don't hardcode the values. Get the value from the playbook and make default as 20...